<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>全般</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>全般</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="0">
             <widget class="QCheckBox" name="chkConnectOnBoot">
              <property name="text">
               <string>起動と同時にブリッジに接続する</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="chkExecTargetAfterWriting">
              <property name="text">
               <string>書き込み完了後ターゲットを起動する</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_4"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ブリッジ</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>シリアルポート</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="1">
               <widget class="QComboBox" name="cmbBaudRate"/>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="cmbSynchMode"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>通信速度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>COMポート　　　　　</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>通信方式</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="cmbComPort"/>
              </item>
              <item row="2" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>SPIモードディレイ</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>マイクロ秒/bit</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="edtSPIDelay"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>ターゲット</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>ブリッジ-ターゲット間通信</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>改行コード</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cmbNewLine"/>
              </item>
              <item row="0" column="3">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>ピン</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_18">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0">
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>PORTB,2</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="cmbPB1Init"/>
            </item>
            <item row="3" column="3">
             <widget class="QComboBox" name="cmbPB2Init"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="edtPinName0"/>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="cmbPB0Func"/>
            </item>
            <item row="3" column="2">
             <widget class="QComboBox" name="cmbPB2Func"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>名称</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>機能</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QComboBox" name="cmbPB0Init"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="edtPinName3"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>PORTB,0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>PORTB,1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QComboBox" name="cmbPB1Func"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>PORTB,3</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="edtPinName5"/>
            </item>
            <item row="4" column="3">
             <widget class="QComboBox" name="cmbPB3Init"/>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>初期値</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QComboBox" name="cmbPB3Func"/>
            </item>
            <item row="1" column="4">
             <widget class="QComboBox" name="cmbPB0Pushed"/>
            </item>
            <item row="2" column="4">
             <widget class="QComboBox" name="cmbPB1Pushed"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="edtPinName4"/>
            </item>
            <item row="4" column="4">
             <widget class="QComboBox" name="cmbPB3Pushed"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="edtPinName2"/>
            </item>
            <item row="3" column="4">
             <widget class="QComboBox" name="cmbPB2Pushed"/>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>押下時</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="edtPinName1"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>DSR</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>CTS</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>読み取りインターバル</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtReadInterval">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>ミリ秒</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="btnOKCancel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
